### PR TITLE
Only exit with code > 0 if the logged message was above notice

### PR DESCRIPTION
**Pull request by bcremer**
This allows to run `bin/openapi` on code that emits errors of type E_DEPRECATED.

Deprecated code should not result in a failing tool, see https://wouterj.nl/2021/11/deprecations-are-not-errors

### CODE REVIEW

File: src/Loggers/ConsoleLogger.php [hunk 1]
```diff
@@ -57,6 +71,10 @@ public function log($level, $message, array $context = []): void
         }
         $stop = !empty($color) ? static::COLOR_STOP : '';
 
+        if (!in_array($level, self::UNCRITICAL_LOG_LEVELS, true)) {
```
Comment (bcremer) [inline src/Loggers/ConsoleLogger.php:74]: I don't like the current naming of `UNCRITICAL_LOG_LEVELS` and `loggedError`.

Will rename to `containsMessageAboveNotive`.

Comment (bcremer) [reply src/Loggers/ConsoleLogger.php]: done

**Comment by DerManoMann** [comment]
Is there any value in keeping the `called` property and method? The only (bad) reason is to keep track of the final processing status and that use case is now gone..

**Comment by bcremer** [comment]
`called()` is still used to generate an extra blank line between log output and the command output: 
https://github.com/zircote/swagger-php/blob/4.2.9/bin/openapi#L216-L218

I would be fine with removing that "feature" to get rid of the extra property.

**Comment by DerManoMann** [comment]
Me too :)

**Comment by bcremer** [comment]
> Is there any value in keeping the called property and method?

And it's gone now.

**Comment by DerManoMann** [comment]
Thanks @bcremer.

This logger is a strange beast. I really would like to separate the logging from the error handling / return value logic but at the time that was more than I was prepared to do ...

**Pull request merged**